src/core/utils/assets.ts

```diff
@@ -83,7 +84,11 @@ export function parseAsset({
 }): ParsedAsset {
   const address = asset.asset_code;
   const chainName = asset.network ?? ChainName.mainnet;
-  const chainId = chainIdFromChainName(chainName);
+  const chainId =
+    chainIdFromChainName(chainName) ||
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    (Number(Object.keys(asset.networks)?.[0]) as ChainId);
```

Comment:
Backend isn't returning a chainId for these assets but the chain id was available under the networks dict.